libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -458,6 +474,36 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         }
         return;
     }
+
+    /**
+     * @hidden
+     * if `columnLayoutType` is given, set those column layouts appropriately
+     */
+    private _setUserSpecifiedLayout(groupField: GroupField): void {
+        const layoutString = this.columnLayoutType.split('-');
+        const xlColumnsNumber = parseInt(layoutString[0].slice(2, layoutString[0].length), 10);
+        const lgColumnsNumber = parseInt(layoutString[1].slice(1, layoutString[1].length), 10);
+        const mdColumnsNumber = parseInt(layoutString[2].slice(1, layoutString[2].length), 10);
+
+        if (isNaN(xlColumnsNumber) || isNaN(lgColumnsNumber) || isNaN(mdColumnsNumber)) {
+            throw new Error('Input a valid number for columns');
+        } else if (xlColumnsNumber > 12 || lgColumnsNumber > 12 || mdColumnsNumber > 12) {
+            throw new Error('Columns cannot be more than 12');
+        } else {
+            const lgColumns = 12 / lgColumnsNumber;
+            const mdColumns = 12 / mdColumnsNumber;
+            const xlColumns = 12 / xlColumnsNumber;
+
+            // for `lg` single-column layout, Styles does not use any class, and providing `fd-col-lg--12` has unintended side-effects
+            // therefore, we remove the lg class for single-column layout
+            if (lgColumns === 12) {
+                groupField.styleClass = `fd-col-xl--` + xlColumns + ` fd-col-md--` + mdColumns;
```

Comment:
Please use one styles for using variable in string
```
`fd-col-xl--${xlColumns} fd-col-md--${mdColumns}`

Reply:
fixed as suggested